sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -486,7 +483,7 @@ case class Factorial(child: Expression) extends UnaryExpression with ImplicitCas
       > SELECT _FUNC_(1);
        0.0
   """)
-case class Log(child: Expression) extends UnaryLogExpression(math.log, "LOG")
+case class Log(child: Expression) extends UnaryLogExpression(StrictMath.log, "LOG")
```

Comment:
Maybe we should modify math.exp to StrictMath.exp in line 361:
# on aarch64
scala> StrictMath.exp(1.0)
res4: Double = 2.7182818284590455

scala> math.exp(1.0)
res5: Double = 2.7182818284590455

# on x86_64:

scala> math.exp(1.0)
res44: Double = 2.718281828459045

scala> StrictMath.exp(1.0)
res45: Double = 2.7182818284590455

Reply:
OK will do.